src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue

```diff
@@ -147,6 +154,53 @@ export default defineComponent<DashboardBoardListProps>({
             selectedId: undefined as string|undefined,
         });
 
+        const cloneModalState = reactive({
+            visible: false,
+            dashboardConfig: {} as DashboardConfig,
+        });
+
+        /* song-lang */
+        const convertBoardItemButtonSet = (dashboardItem) => [
```

Comment:
suggestion: please specify types!

```suggestion
        const convertBoardItemButtonSet = (dashboardItem: ??): IconSet[] => [
```